subsys/bluetooth/controller/hci/hci.c

```diff
@@ -3610,13 +3610,14 @@ static void le_scan_req_received(struct pdu_data *pdu_data,
 
 	/* TODO: fill handle when Adv Ext. feature is implemented. */
 
-	if (!(event_mask & BT_EVT_MASK_LE_META_EVENT) ||
+	if (!IS_ENABLED(CONFIG_BT_CTLR_ADV_EXT) ||
```

Comment:
Revert this change/commit. Because there is code inside this if-clause, instead callee be conditionally compiled. As this is something that could be done for other features, lets not do this in this PR.

Suggest to remove this commit from this PR.